local-interchain/interchain/start.go

```diff
@@ -98,7 +115,7 @@ func StartChain(installDir, chainCfgFile string, ac *types.AppStartConfig) {
 
 	chains, err := cf.Chains(testName)
 	if err != nil {
-		log.Fatal("cf.Chains", err)
+		logger.Fatal("cf.Chains", zap.Error(err))
```

Comment:
It would be awesome to find a way to enforce our logging style guide via the linter. I'm sure the codebase has many inconsistencies at this point from people not remembering about the style guide during code review but figured i would point it out since we have been putting some effort into consistency across the codebase.